recipes-multimedia/libcamera-apps/libcamera-apps.bb

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Raspberry libcamera-apps"
+
+LICENSE = "BSD-2-Clause"
+LIC_FILES_CHKSUM = "file://license.txt;md5=a0013d1b383d72ba4bdc5b750e7d1d77"
+
+SRC_URI = "git://github.com/raspberrypi/libcamera-apps.git;branch=main;protocol=https"
+
+PV = "1.0+git${SRCPV}"
+SRCREV = "6145daf735fcb0a6e26699a407fe80307c24f6db"
+
+S = "${WORKDIR}/git"
+
+DEPENDS = "libcamera opencv boost libjpeg-turbo libexif tiff"
+
+inherit cmake pkgconfig
+
+# Specify extra options:
+# EXTRA_OECMAKE = "-DENABLE_DRM=0 -DENABLE_X11=0 -DENABLE_QT=0 -DENABLE_OPENCV=1 -DENABLE_TFLITE=0"
```

Comment:
Clean up this comment.

Reply:
Remove it you mean?
Letting user know options to bbapend and customize the build makes sense as a comment?

Reply:
I'm not sure what this is all about. Could you explain what did you mean by this comment block?

Reply:
This are cmake options for libcamera-apps, that are documented here:
https://www.raspberrypi.com/documentation/accessories/camera.html#building-libcamera-apps

We can check automatically for this options, if opencv, tensorflow, qt, x11 are present, and -DENABLE_COMPILE_FLAGS_FOR_TARGET=armv8-neon when building for Pi 3 and Pi 4 in 32 bit.

Reply:
That's what we should do. Based on distro features.